<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QMainWindow" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>658</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Отрезки</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="mainview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>511</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="what">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="circle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Окружность </string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ellips">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Эллипс </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="other">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>411</width>
      <height>641</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSpinBox" name="a">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>370</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>511</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QRadioButton" name="middle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Средней точки</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="bground_color">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>170</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="lib">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Библиотечный</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="step">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>440</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="angle_step">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>470</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Количество </string>
     </property>
    </widget>
    <widget class="QSpinBox" name="rad">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>310</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>511</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QPushButton" name="draw_once">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>400</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Отобразить фигуру</string>
     </property>
    </widget>
    <widget class="QPushButton" name="draw_centr">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>500</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Концентрические фигуры</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="param">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Параметрическое уравнение </string>
     </property>
    </widget>
    <widget class="QSpinBox" name="b">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>370</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>511</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="begin_point_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>A</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="brez">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Брезенхем</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="centr_x">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>511</number>
     </property>
     <property name="value">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="count">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>470</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
    </widget>
    <widget class="QLabel" name="begin_point_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>370</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>420</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="centr_y">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>240</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>511</number>
     </property>
     <property name="value">
      <number>255</number>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выбрать цвет линии</string>
     </property>
    </widget>
    <widget class="QLabel" name="begin_point">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Радиус </string>
     </property>
    </widget>
    <widget class="QLabel" name="end_point">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>240</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Центр</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="line_color">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>200</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_bground">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выбрать цвет фона</string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>540</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="begin_point_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Для окружности:</string>
     </property>
    </widget>
    <widget class="QLabel" name="begin_point_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>340</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Для эллипса:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clean_all">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>560</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Очистить холст</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="canon">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Каноническое уравнение </string>
     </property>
    </widget>
    <widget class="QLabel" name="diametr">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>440</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Шаг </string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="time_cmp">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>570</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Временные характеристики</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
